Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -325,7 +325,7 @@ int TextEditor::ruler_width() const
 {
     if (!m_ruler_visible)
         return 0;
-    int line_count_digits = static_cast<int>(log10(line_count())) + 1;
+    int line_count_digits = static_cast<int>(AK::log10((double)line_count())) + 1;
```

Comment:
Prefer log10<double>(..) over log10((double)..)...

Reply:
idk if it'll cast imlplicitly then, I'll go try it